### ISSUE TITLE
Certificate Problem when starting a test

### ISSUE THREAD

**Issue by reply1991** [chaos-mesh/chaos-mesh#3314]
Deployed with helm on AKS. all pods up&running and all resources created.

I try then to create a crd NetworkChaos as in the official documentation but I get a x509 signed by unknown authority when trying to contact the Controller Manager svc. the certificate used is stored in the webhook-certs automatically generated by the helm.

Can anyone help me? Should I put a trusted certificate instead?

**Comment by STRRL** [comment]
Hi! Could you provide the following informations for us?

- did you operate upgrade/uninstall before? 
- do you use Chaos Mesh with a service mesh framework(like istio) which might intercept the network traffic?
- the version of Chaos Mesh
- the version of kubernetes
- the log of pod chaos-daemon-xxx() contains "x509 errors"

**Comment by STRRL** [comment]
> actually it's not the chaos-daemon logging the error, it's when I try to create the CRD NetWorkChaos

I think you have enabled the "istio sidecar pod injection" and "TLS", istio would "wrap" the original traffic with its TLS layer. 

Chaos Mesh would register “Validating Admission Webhooks/Mutating Admission Webhook” into kubeapiserver, for 
validating/updating CRDs like `IOChaos`, `NetworkChaos`.

You could use this annotation on Pod "chaos-controller-manager-***" to avoid it:
```
annotations:
        traffic.sidecar.istio.io/excludeInboundPorts: "9443"
```

refer: 

- https://istio.io/latest/zh/docs/reference/config/annotations/
- https://cloud-native.slack.com/archives/C0193VAV272/p1647601003534599?thread_ts=1647549411.867719&cid=C0193VAV272

**Comment by reply1991** [comment]
Hello @STRRL thank you for your answer.

Unfortunately your solution did not work. I also opened an interactive terminal on daemon pod and tried to post the svc (as said in the log) and I replicated the error. I tried then with -k to skip ssl and it went successfully.

I then used openssl and the certificate retrieved from the server is the name shown in the certificate. I think istio is not intercepting / rewriting anything here.

Many thanks again
